com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/BloomPrefilter.compute

```diff
@@ -7,17 +7,34 @@
 
 #pragma kernel KMain
 
+#pragma multi_compile LOW_QUALITY HIGH_QUALITY
+
 TEXTURE2D_X(_InputTexture);
 
 RW_TEXTURE2D_X(float3, _OutputTexture);
 
 SAMPLER(sampler_LinearClamp);
+SAMPLER(sampler_BilinearClamp);
```

Comment:
linear sampler is bilinear 

Also, we should use the pre-declared samplers (i.e. s_point_clamp_sampler)